esp/scm/ws_workunits.ecm

```diff
@@ -2186,7 +2186,7 @@ ESPservice [
     ESPmethod [cache_seconds(30)] WUFullResult(WUFullResultRequest, WUFullResultResponse);
```

Comment:
Where did these timeouts values come from?  For instance I think graph stats update every 30s or so in thor.  If there is an additional 30s cache it may mean that the user now has to wait twice as long for them to be updated.  (Unless the UI can explicitly request a refresh or lower cache time when the user presses refresh.)